Clarifying "Finance" section as working through payroll (#36)
hyphacoop/handbook: finance.md

### Reporting & Remitting Taxes

> Hypha has the following [_program accounts_][program-accounts] with the Canada Revenue Agency (CRA):

Reviewer: ```suggestion
Hypha has the following [program accounts][program-accounts] with the Canada Revenue Agency (CRA):
```

   [program-accounts]: https://www.canada.ca/en/revenue-agency/services/tax/businesses/topics/changes-your-business/adding-accounts-your-business-number-bn.html

- :ballot_box_with_check: `RC0001` **Corporate Income Tax (RC)** is filed/reported and remitted/paid each tax year

> - :soon: `RP0001` **Payroll Deduction (RP)** is reported and paid each pay period

Author: ```suggestion
- :ballot_box_with_check: `RP0001` **Payroll Deduction (RP)** is reported and paid each pay period
```

- :soon: `RT0001` **GST/HST (RT)** is [reported and paid][sales-tax-how] each filing period (TBD)

>    [sales-tax-about]: https://www.canada.ca/en/revenue-agency/services/tax/businesses/topics/gst-hst-businesses/account-register.html#once

Author: ```suggestion
   [sales-tax-how]: https://www.canada.ca/en/revenue-agency/services/tax/businesses/topics/gst-hst-businesses/account-register.html#once
```
